Files/Views/LayoutModes/GenericFileBrowser.xaml.cs

```diff
@@ -436,9 +440,26 @@ protected override ListedItem GetItemFromElement(object element)
             return row.DataContext as ListedItem;
         }
 
-        private void RightClickContextMenu_Opening(object sender, object e)
+        [DllImport("user32.dll", CharSet = CharSet.Auto, ExactSpelling = true)]
```

Comment:
iirc user32.dll is not available to UWP apps.

Reply:
It is [available](https://docs.microsoft.com/en-us/uwp/win32-and-com/win32-extension-apis#apis-from-user32dll), but there is no method which is used in this PR.

Reply:
@tsvietOK You're right, it's just very limited.

Reply:
Thank you; you are both correct. I should have used

```
private readonly Windows.UI.ViewManagement.UISettings uiSettings = new Windows.UI.ViewManagement.UISettings();
uiSettings.DoubleClickTime
```

I'm working on a new version of this fix with a couple of other small tweaks to the code and then I'll update the PR in a few days.

Reply:
@rjnb50 You can use `GetDoubleClickTime()` from Vanara library